### PR TITLE
Fix root

### PR DESCRIPTION
**Pull request by baronfel**
Fixes at least part of #46 by setting the expected content root, since this site it served at a subpath of `fsharp.github.io`.

In addition, I tried to correct the bad urls in `index.md` by adding the `{{root}}` parameter for replacement, but that did not work. F#F probably is missing something here.

### CODE REVIEW

**Comment by baronfel** [comment]
As an aside, I noticed while trying to work this that F#F parses users parameters pairwise. Is that a meaningful ordering? people don't usually send parameters expected the space-delimited list to be traversed in that way. I could see windows of 2, or requiring `<key>=<value>` syntax, but the way they're parsed right now is confusing to me.

**Comment by baronfel** [comment]
@dsyme I saw you just did a batch of fixes to F#F, can you check this change too? I think this will be required anyway, since we don't have the site as `/` in this repo.

**Comment by dsyme** [comment]
> @dsyme I saw you just did a batch of fixes to F#F, can you check this change too? I think this will be required anyway, since we don't have the site as / in this repo.


Yes this was an F#F bug , should be fixed

**Comment by dsyme** [comment]
> As an aside, I noticed while trying to work this that F#F parses users parameters pairwise. Is that a meaningful ordering?

agreed this is weird. Inherited it from previous code

**Comment by dsyme** [comment]
> Fixes at least part of #46 by setting the expected content root, since this site it served at a subpath of fsharp.github.io.

The mistake was that PackageProjectUrl is the correct value for root

File: .github/workflows/docs.yml [hunk 1]
```diff
@@ -33,7 +33,7 @@ jobs:
     - name: Build FSharp.Formatting master
       run: cd FSharp.Formatting && .\build -t Build
     - name: Run fsdocs
-      run: FSharp.Formatting\src\FSharp.Formatting.CommandTool\bin\Release\netcoreapp3.1\fsdocs.exe build --sourcefolder fsharp  --parameters  fsdocs-navbar-position fixed-left
+      run: FSharp.Formatting\src\FSharp.Formatting.CommandTool\bin\Release\netcoreapp3.1\fsdocs.exe build --sourcefolder fsharp  --parameters  fsdocs-navbar-position fixed-left root /fsharp-core-docs/
```
Comment (dsyme) [inline .github/workflows/docs.yml:36]: Basically, if F#F isn't doing the right thing for this repo without parameters, it's a bug in F#F :-)

**Comment by baronfel** [comment]
Great! Sounds like I can close then then.

**Pull request closed without merging**